underscore.js

```diff
@@ -20,6 +20,7 @@
 
   // Save bytes in the minified (but not gzipped) version:
   var ArrayProto = Array.prototype, ObjProto = Object.prototype;
+  var SymbolProto = typeof Symbol !== 'undefined' ? Symbol.prototype : null;
```

Comment:
I donno how much of a concern this is but the `undefined` check alone could let a `null` value through and cause a script error at that point.